routes/api/events.js

```diff
@@ -1,9 +1,376 @@
 const express = require('express');
 const router = express.Router();
+const shortid = require('shortid');

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/spumony/xvent-project-backend/pullRequest?prid=7531905)
```suggestion
const shortid = require("shortid");
```